cmd/release-controller/http_candidate.go

```diff
@@ -106,7 +106,13 @@ func (c *Controller) httpReleaseCandidateList(w http.ResponseWriter, req *http.R
 	vars := mux.Vars(req)
 	releaseStreamName := vars["release"]
 	successPercent := 80.0
-	releaseCandidateList, err := c.findReleaseCandidates(successPercent, releaseStreamName)
+	promote := true
+	minUpgradeEdges := 2
```

Comment:
This doesn't seem right @jwforres.  I don't think we care about number of upgrade edges, just whether they exist to the releases we care about (latest 4.1.z, etc).  It's not about quantity, but diversity.

Reply:
Let's separate this out and do it after we get your base endpoint merged.  I don't want to block this merging on arguing about this change.

Reply:
We hadn't discussed heuristics yet. Agreed that a raw number of passed upgrades doesn't have much meaning.

I do care if we have have spun up upgrades across 10 different versions and only a couple of them passed, which sounds more like the successPercent.

Absolutely agree lets factor out how the heuristics are applied, we'll want that done anyway to make it easy to update the rules in the future.

Reply:
Reverted